apps/wing-console/console/server/src/utils/createRouter.ts

```diff
@@ -120,6 +125,22 @@ const invalidateQueriesAfterMutation = middleware(async (options) => {
   return result;
 });
 
-export const createProcedure = t.procedure.use(
-  invalidateQueriesAfterMutation,
-) as typeof t.procedure;
+const sendUserResourceInteractionAnalyticsEvent = middleware(
+  async (options) => {
+    const analytics = options.ctx.analytics;
+    if (analytics) {
+      const meta = options.meta;
+      if (meta?.resource && meta?.action) {
+        analytics.track("user_resource_interaction", {
+          resource: meta.resource,
+          action: meta.action,
+        });
+      }
+    }
+    return await options.next();
+  },
+);
```

Comment:
Instead of using placing `meta` properties on every route, maybe we could use `options.path`?

Reply:
NVM, I think this is better: https://github.com/winglang/wing/pull/5795/files#r1506168813